SalesforceCommerceCloud/pwa-kit
# [App-extensibility] W-17161438: Extract const values into extension config

# Description
This PR attempted to move as many as constant values into extension config. Some constants are not possible to move yet, e.g translation constants. The idea is to unify all of our configuration into one single place



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

-  Checkout the PR
- Add storefront extension into typescript-minimal
- npm i
- start the typescript minimal app
- You should see the retail app showing up and functional as normal
- Adjusting the config values should reflect on the running app

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
